src/server.js

```diff
@@ -9,6 +10,9 @@ const app = express();
 app.use(morgan(logLevel));
 app.use(express.json());
 app.use('/v1', v1Router);
+app.use(cors({
+  origin: 'http://localhost:3001',
```

Comment:
store the string as a constant called `corsHost` in ` config/index.js` similar to `dbUrl`, `port`, etc.
No hardcoded strings in the `server.js` file, we need them all to be overwrite-able via an environment variable.